php-frontend/src/main/java/org/sonar/php/symbols/UnknownMethodSymbol.java

```diff
@@ -48,4 +48,9 @@ public Trilean isOverriding() {
   public Trilean isAbstract() {
     return Trilean.UNKNOWN;
   }
+
+  @Override
+  public ClassSymbol owner() {
+    return null;
```

Comment:
I think I would prefer to return something else, maybe an `UnknownClassSymbol`.
Although I forgot the purpose of this `UnknownMethodSymbol`...

Reply:
`return new UnknownClassSymbol(QualifiedName.qualifiedName("unknown"));`
This would be the unknown symbol.